docs/faq.rst

```diff
@@ -44,6 +44,16 @@ Both Kolibri and `Kolibri Studio <https://studio.learningequality.org/>`_ are tr
 +---------------------------+-----------------+-----------------+
 | Hindi                     | ✓               | ⨯               |
 +---------------------------+-----------------+-----------------+
+| Tamil                     | ✓               | ⨯               |
++---------------------------+-----------------+-----------------+
+| Telugu                    | ✓               | ⨯               |
++---------------------------+-----------------+-----------------+
+| Marathi                   | ✓               | ⨯               |
++---------------------------+-----------------+-----------------+
+| Chinyanja                 | ✓               | ⨯               |
++---------------------------+-----------------+-----------------+
+| Portuguese (Brasil)       | ✓               | ⨯               |
++---------------------------+-----------------+-----------------+
```

Comment:
I'm not sure what "(✓ ver 0.5)" means. Maybe we should delete that?

Also in the [changelog](https://github.com/learningequality/kolibri/blob/develop/CHANGELOG.rst#092), I've started using the ordering convention of: English first, everything else alphabetical

Reply:
Kolibri was localized into Mexican Spanish for version 0.5 last year (UNETE launch), but that was never updated. As per Lauren's suggestion, I left only the check marks, but I missed the suggestion about ordering convention - will amend!